workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-organization-select/index.js

```diff
@@ -269,6 +269,7 @@ function filterRemoteOptions(query, abortController) {
 	}
 
 	const url = new URL(input.attributes.relationshipURL);
+	url.searchParams.set('sort', 'name:asc');
```

Comment:
@JoMaAlves can we move this line down 1 line to have it sorted alphabetically? If it needs to come first that is fine, we would just need to add another line break after the line

Reply:
Sure, no problem!